audformat/core/database.py

```diff
@@ -465,6 +498,12 @@ def update(
                 same ID is found
             ValueError: if table data cannot be combined (e.g. values in
                 same position overlap)
+            RuntimeError: if ``copy_media=True``,
+                but one of the databases in ``others``
+                was not saved (contains files but no root folder)
+            RuntimeError: if any involved database is not portable
```

Comment:
Maybe simplify to just "if any involved database is not portable"?

Reply:
Ok, it's just that with the following "and" it sounds like `portable` and containing absolute paths etc. are two different things.

Reply:
Yes, the `and` makes it indeed hard. We could replace it by something `which means`, but I have no deleted it as it is also the safer option if we at some point change what `is_potable()` is checking the docstring here is still correct.